### PR TITLE
fixes: design changes 

### PR DESCRIPTION
**Pull request by mul53**
- add design changes for home, swap, pool, bridge and farm pages

### CODE REVIEW

File: src/components/HomeCards/prices.tsx [hunk 1]
```diff
@@ -1,155 +1,31 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import styled from 'styled-components'
-import { useSpring, animated } from 'react-spring'
+import numeral from 'numeral'
```
Comment (leonprou) [inline src/components/HomeCards/prices.tsx:3]: How this is used?

File: src/components/farm/FarmList/index.tsx [hunk 2]
```diff
@@ -1,35 +1,47 @@
 import React, { useState } from 'react'
-import { Flex } from 'rebass'
+// import { Flex } from 'rebass'
```
Comment (leonprou) [inline src/components/farm/FarmList/index.tsx:2]: please delete comment

File: src/components/farm/FarmList/index.tsx [hunk 3]
```diff
@@ -1,35 +1,47 @@
...
-  text-align: left;
-  line-height: 3rem;
+const Th = styled.th`
+  border-bottom: 1px solid #111219;
+  padding 12px 16px;
+  font-weight: 500;
 `
 
-export default function FarmList({ farms }: any) {
+// const Header = styled.div`
```
Comment (leonprou) [inline src/components/farm/FarmList/index.tsx:27]: here as well

File: src/components/farm/FarmListItem/index.tsx [hunk 4]
```diff
@@ -1,143 +1,124 @@
-import React, { useState } from 'react'
+import React, { useCallback } from 'react'
 import styled from 'styled-components'
 import Icon from '../FarmList/icons'
-import { Flex } from 'rebass'
+// import { Flex } from 'rebass'
```
Comment (leonprou) [inline src/components/farm/FarmListItem/index.tsx:4]: and here?

**Review by leonprou — APPROVED** [review]

File: src/components/HomeCards/prices.tsx (hunk 1 above)
Comment (mul53) [reply src/components/HomeCards/prices.tsx:3]: it's used for formatting numbers, so instead of 1000000 it can format to 1M

**Review by leonprou — APPROVED** [review]

**Pull request merged**